nxviz/plots.py

```diff
@@ -301,6 +307,14 @@ def compute_edge_colors(self):
             )
             self.sm._A = []
 
+    def compute_edge_widths(self):
```

Comment:
Everything in the PR looks good.

I'm wondering if you might be open to contributing a test for `compute_edge_widths()`? One idea might be to use the graph you created in one of the examples, and then simply test that the plot's `.edge_widths` attribute is set correctly, perhaps? The example is good, but having an automated test written makes the software just a bit more robust.